qctrlopencontrols/driven_controls/driven_control.py

```diff
@@ -544,21 +603,22 @@ def export(
         self, coordinates=Coordinate.CYLINDRICAL.value, dimensionless_rabi_rate=True
     ):
 
-        """ Returns a dictionary formatted for plotting using the qctrl-visualizer package.
+        """
+        Returns a dictionary formatted for plotting using the ``qctrl-visualizer`` package.
 
         Parameters
         ----------
-        dimensionless_rabi_rate: boolean
-            If True, normalizes the Rabi rate so that its largest absolute value is 1.
         coordinates: string
             Indicates whether the Rabi frequency should be plotted in terms of its
-            'cylindrical' or 'cartesian' components.
+            'cylindrical' or 'cartesian' components. Defaults to 'cylindrical'.
+        dimensionless_rabi_rate: boolean
```

Comment:
I think this might be my mistake when writing this docstring, but maybe these parameters are optional

Reply:
Yup, done.